packages/ui-react/src/containers/ShelfList/ShelfList.tsx

```diff
@@ -24,6 +28,38 @@ type Props = {
   rows: Content[];
 };
 
+const useEmptyPlaylistsReporter = (rows: Content[]) => {
+  const queryClient = useQueryClient();
+  const [isEmpty, setEmpty] = useState(false);
```

Comment:
Small suggestion: let's rename the setter to setIsEmpty for consistency with the isEmpty state.